dist/svg.panzoom.js

```diff
@@ -22,25 +22,26 @@ SVG.extend(SVG.Doc, SVG.Nested, {
   panZoom: function(options) {
     options = options || {}
     var zoomFactor = options.zoomFactor || 0.03
-
     var zoomMin = options.zoomMin || 0
```

Comment:
The default value should be something bigger than `0` since `0` will guarantee `Inifinite` in `zoomAmount`. I think `Math.MIN_VALUE` is a better default